examples/demo/src/layout/themes.ts

```diff
@@ -6,7 +6,7 @@ export const darkTheme = {
         secondary: {
             main: '#FBBA72',
         },
-        type: 'dark', // Switching the dark mode on is a single property value change.
+        type: 'dark' as 'dark', // Switching the dark mode on is a single property value change.
```

Comment:
:scream: :thinking:

Reply:
Does it mean users will have to cast this as well ?

Reply:
yes, that's because MUI defines the `type` type as `'dark' | 'light'`, and TypeScript is dumb. Nothing we can do to avoid that.